### ISSUE TITLE
drush8 for D7&D8

### ISSUE THREAD

**Issue by tlacoere** [drush-ops/drush#3535]
Dear,

First I want to appologise, I don't have the technical knowledge to understand half of the threads here. I know how to configure and build using Drupal and I know how to set up a LAMP stack. When I began with Drupal a decade ago I understood that was all I had to know. These days, however, I feel very much lost and the simple questions I have seem to be answered differently across the web.

With the recent core update I installed drush8 on all of my websites and updated 24 cores, both D7 and D8, using drush 8.1.16. As Drupal seems to release new cores more frequently than before (and since I was around for Drupalgeddon) this was the first time using Drush on all of them to save time. But I did read the instructions, apparently I should have used Drush9 for Drupal8 updates.

First question: **can existing Drupal8 websites be updated using Drush8**? It seems I didn't run into any problems doing so, but it also seems some say it's a bad idea. Why? Did I damage my websites doing so? Here's what's on Drupal.org

> if you are a long-time Drush user, you may still use Drush 8 (8.14 or newer) to update Drupal 8

This information is either correct, in which case it should be pointed out on the drush docs that using drush8 is fine, or it isn't in which case Drupal has to update this section. When reading the following (they use dev8.5 in the example) I'm assuming these are all drush8 compatible commands: https://www.drupal.org/docs/8/update/update-core-via-drush

Second question: **how do I use drush9 on an existing website**? Actually, I did look this one up and the process seems very cumbersome. Most of my D8 websites are quite small and from what I understand it's best to start from scratch again... What I found most usefull in drush was that it allows you to make a complete code dump and it stores updated files in the drush backups directory. So let me rephrase the question: **Will drush9 and composer ever reintroduce the backup functionality in the future?** The docs state: _archive-dump has unfortunately been removed from Drush 9, so you'll have to use a standard CLI tool to backup the needed directories - typically excluding the vendor directory._ What CLI tool? Why did this functionality not make it into 9?

Story of my life (I didn't write it, but some will relate) can be found here: https://www.drupal.org/forum/support/post-installation/2017-03-24/how-do-i-migrate-existing-d8-site-into-a-composer-managed#comment-12328927

Thanks to anyone able to help me and many others keeping their websites safe,
Sincerely,
A long time Drupal user who _simply wants to update a website_ :)

**Comment by weitzman** [comment]
Please use Drupal Answers for Support.
